src/plugins/intel_cpu/src/nodes/tensoriterator.cpp

```diff
@@ -505,14 +562,18 @@ void TensorIterator::prepareParams() {
     before_mappers.clear();
     back_mappers.clear();
 
+    buffers.clear();
```

Comment:
I'm wondering, do we really need to reinstantiate the buffers? M.b. it does make sense to just reset the `max_iterations` value? These memory reallocations for all the auxiliary data are not free of charge.

Reply:
updated.